Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -141,9 +141,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
         fragmentManager.beginTransaction().replace(R.id.inc, fragment).commit();
 
-        // Get runtime permissions for Android M
```

Comment:
why did you remove these? I didnt get the point for this

Reply:
I removed these permissions because they will trigger the permissions to be asked if the user is just going to log-in or sign-up. I wanted the permissions to be only used at activities/app sections where it is only necessary. 

@thedevelopersanjeev 's implementation could be better but it's already asking for permissions at first time use for user and it doesn't take into account if the user declines all permissions.